packages/nimble-components/src/card/styles.ts

```diff
@@ -21,4 +23,13 @@ export const styles = css`
         color: ${bodyFontColor};
         background-color: ${sectionBackgroundColor};
     }
+
+    ::slotted([slot='title']) {
+        font: ${titleFont};
+        color: ${titleFontColor};
+    }
+
+    section {
```

Comment:
There aren't many styles yet but get used to organizing styles in document order: https://github.com/ni/nimble/blob/main/packages/nimble-components/docs/css-guidelines.md#group-selectors-by-target-and-document-order

![image](https://github.com/ni/nimble/assets/1588923/ca166517-71f2-43cd-a39e-e5d25c3df5bb)

Reply:
Moved the section style above the slot. Thanks for the link!